js/gated-content/src/views/BlogPage.vue

```diff
@@ -5,10 +5,13 @@
     </div>
     <div v-else-if="error">Error loading</div>
     <template v-else>
-      <div class="blog-page__image" v-bind:style="{
-              backgroundImage: `url(${image})`
-      }">
+      <div v-if="blog.attributes.field_vy_blog_image !== null">
```

Comment:
@bharath-kondeti is it possible to put this `v-if` on the div below so we're not creating unnecessary divs?

So, something like `<div class="blog-page__image" v-if="blog.attributes.field_vy_blog_image !== null" v-bind:...`